### PR TITLE
Cards for some SMP processes (for a WZ Run3 measurement)

### PR DESCRIPTION
**Pull request by Cvico**
Hi,
 
We are interested in measuring WZ with Run3 data, and we would need some MC samples 
that are not centrally handled. I've been told by the SMP MC contacts that I have to submit 
the cards for the processes I'm interested to produce. 

This PR includes the cards for:

- WZTo3LNu (aMC@NLO)
- ZZ (powHeg)
- WG and ZG (aMC@NLO)

I've been trying to be consistent with Run2 settings (except for center of mass energy, which I have changed
to match Run3 conditions).

I would like to have them reviewed and included in the main repository once approved.

Best regards,
Carlos Vico

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZTo3LNu01j_5f_NLO_FXFX/WZTo3LNu01j_5f_NLO_FXFX_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+ 15  = bwcutoff
+#***********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+   1  = jetalgo   ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+ 1.0  = jetradius ! The radius parameter for the jet algorithm
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZTo3LNu01j_5f_NLO_FXFX/WZTo3LNu01j_5f_NLO_FXFX_run_card.dat:116]: Hi, could you please do a comparison between "1.0  = jetradius" and "0.7  = jetradius' while keeping all the others the same, e.g., on the total cross section, and kinematic distribution of leptons

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZGTo2LG_01j_5f_NLO_FXFX/ZGTo2LG_01j_5f_NLO_FXFX_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model loop_sm-no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZGTo2LG_01j_5f_NLO_FXFX/ZGTo2LG_01j_5f_NLO_FXFX_proc_card.dat:11]: these lines could be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZGTo2LG_01j_5f_NLO_FXFX/ZGTo2LG_01j_5f_NLO_FXFX_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,17 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZGTo2LG_01j_5f_NLO_FXFX/ZGTo2LG_01j_5f_NLO_FXFX_proc_card.dat:4]: these lines could be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZGTo2LG_01j_5f_NLO_FXFX/ZGTo2LG_01j_5f_NLO_FXFX_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+#***********************************************************************
+# Maximal PDG code for quark to be considered a jet when applying cuts.*
+# At least all massless quarks of the model should be included here.   *
+#***********************************************************************
+ 5 = maxjetflavor
+#***********************************************************************
+# For aMCfast+APPLGRID use in PDF fitting (http://amcfast.hepforge.org)*
+#***********************************************************************
+ 0 = iappl   ! aMCfast switch (0=OFF, 1=prepare APPLgrids, 2=fill grids)
+#***********************************************************************
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZGTo2LG_01j_5f_NLO_FXFX/ZGTo2LG_01j_5f_NLO_FXFX_run_card.dat:159]: these lines could be removed

**Comment by sihyunjeon** [comment]
WZ ZZ WG ZG are definitely in the central list

**Comment by Cvico** [comment]
Hi @sihyunjeon, I knew that WZ PowHeg was in the central plans, but also the aMC@NLO production? For the others I didn't knew. Should I remove them from the PR?

**Comment by menglu21** [comment]
thanks @sihyunjeon for the reminder. @Cvico yes, both of them are in the central list, it could be removed

**Comment by Cvico** [comment]
Hi @menglu21, sorry for the late reply. I think we can close the PR then, if that is the case, since ZG WG WZ and ZZ are exactly the content I wanted to add in my PR, and all of them are going to be central according to the GEN spreadsheet. I didn't know that samples with score 2 in that spreadsheet were also handled centrally when I did the PR. I apologise.

**Comment by menglu21** [comment]
no worries, closed.

**Pull request closed without merging**